pkgs/games/runescape-launcher/wrapper.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildFHSUserEnv, runescape-launcher-runtime }:
```

Comment:
Please require package individually here instead of using `pkgs`. Otherwise users cannot use `override` properly.

Reply:
Isn't `buildFHSUserEnv` supposed to be used like this? Could you demonstrate?

Reply:
There isn't any kind of support for comfortable overriding in `buildFHSUserEnv` now. In theory we could add `overrides` argument akin to `haskell.packages` one.